docs/src/languages/dotnet.md

```diff
@@ -12,9 +12,9 @@ description: |
 |----------------------------------|---------------|
 |  {{< image-versions image="dotnet" status="supported" environment="grid" >}} | {{< image-versions image="dotnet" status="supported" environment="dedicated" >}} |
 
-To specify a .NET Core container, use the `type` property in your [app configuration](../configuration/app/app-reference.md).
+{{< image-versions-legacy "dotnet" >}}
```

Comment:
Is it expected that none are displayed?
In some cases, in the registry ` "deprecated": [],`.

Reply:
Not sure what you mean, since this shortcode is about legacy regions. I don't want it displayed if there aren't any differences.